From 5d56cd72450c88c6baef8d5d0bd6e32500dbae81 Mon Sep 17 00:00:00 2001
From: techyminati <techyminati@outlook.com>
Date: Tue, 4 Nov 2025 13:02:20 +0100
Subject: [PATCH] Settings: Remove eSIM Delete Pref from reset options

* We handle this in OpenEUICC app aka eSIM Manager
* This option doesnt work in our case.
* Remove this to avoid user confusion & fix the duplication
* Fix the reset settings crash.

Signed-off-by: techyminati <techyminati@outlook.com>
---
 res/xml/reset_dashboard_fragment.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/xml/reset_dashboard_fragment.xml b/res/xml/reset_dashboard_fragment.xml
index 0fe929772e1..a6ebe85c244 100644
--- a/res/xml/reset_dashboard_fragment.xml
+++ b/res/xml/reset_dashboard_fragment.xml
@@ -45,7 +45,7 @@
     <Preference
         android:key="erase_euicc_data"
         android:title="@string/reset_esim_title"
-        settings:isPreferenceVisible="@bool/config_show_sim_info"
+        settings:isPreferenceVisible="false"
         settings:controller="com.android.settings.network.EraseEuiccDataController" />
 
     <!-- Delete private space -->
-- 
2.49.0

